_config.yml

```diff
@@ -24,3 +24,11 @@ github:
 # Your search_site_handle value is from the search.gov settings page.
 
 # search_site_handle: searchgov-site-handle
+
+# Change the theme variant. Current options are: "beach"
+theme_variant: beach
+sass:
+  load_paths:
+    - _sass
+    - assets/themes
+    - assets/css
```

Comment:
I think typically Sass partials are found in the `_sass` directory - any reason to use `assets/css` over the standard location? If we do want to use `assets/css` I think simplifying this to:
```
load_paths:
  - _sass
  - assets
```
should work (I think it looks into the directory recursively).

Reply:
I didn't mean to commit that `assets/css` - removing in the next commit

Reply:
Does this still need to be changed, @line47?

Reply:
I think it just needs to be `- assets/css`, I'm making that update now 👍